Model/Client.php

```diff
@@ -243,6 +251,9 @@ private function _getApiUrl($resource)
             case 'customers':
                 $apiUrl .= '/customers';
                 break;
+            case 'taxes':
+                $apiUrl .= '/magento/taxes';
+                break;
```

Comment:
@ryanreeves-taxjar Do we still need this code if we're not moving forward with `Model/Client.php` for calculations?

Reply:
@fastdivision All of our api endpoints are defined here.  Even though `/magento/taxes` is (now) unused, I think it makes sense to keep it.